packages/ai/src/server/socket.ts

```diff
@@ -1,5 +1,7 @@
 import { Socket, io } from 'socket.io-client';
 import { logger } from '@rsdoctor/utils/logger';
+import { GlobalConfig } from '@rsdoctor/utils/common';
+import fs from 'fs';
```

Comment:
Adding prefix is better:

```suggestion
import fs from 'node:fs';
```